src/Data/ProgrammesDb/Type/UtcDateTimeType.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace BBC\ProgrammesPagesService\Data\ProgrammesDb\Type;
+
+use DateTimeImmutable;
+use DateTimeInterface;
+use DateTimeZone;
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Types\ConversionException;
+use Doctrine\DBAL\Types\DateTimeType;
+use InvalidArgumentException;
+
+class UTCDateTimeType extends DateTimeType
+{
+    static private $utc;
+
+    public function convertToDatabaseValue($value, AbstractPlatform $platform): ?string
+    {
+        if (!$value instanceof DateTimeInterface) {
+            throw new InvalidArgumentException('$value must implement DateTimeInterface');
+        }
+
+        $utcValue = $value->setTimezone(new DateTimeZone('UTC'));
```

Comment:
Changing this to expect DateTimeImmutable only will wreck merry havoc as Doctrine expects to send and return DateTime (not Immutable). Making that inconsistent with this that default behaviour will cause weirdness.